tests/unittests/OnnxImporterTest.cpp

```diff
@@ -659,6 +659,12 @@ TEST_F(OnnxImporterTest, importDivUniBroadcastOp6Axis) {
       [](float a, float b) { return a / b; });
 }
 
+TEST_F(OnnxImporterTest, importPowMultiBroadcastOp7) {
+  importArithMultiBroadcastTest<PowNode>(
+      "powMultiBroadcastOp7.onnxtxt", {1, 3, 1, 2}, true, 1, 2,
```

Comment:
Bit confused here -- why pass 1 and 2, these are bools right? So just pass true for both then?

Reply:
Right, I rebased this change from an older code base where `importArithMultiBroadcastTest` had a bit different interface and did not notice that. Fixed now.